setup.py

```diff
@@ -92,10 +95,12 @@ def build_extension(self, ext):
             cmake_args += ['-DCMAKE_INSTALL_PREFIX={}'.format(localdir)]
 
             if platform.system() == 'Windows':
-                if self.compiler.compiler_type == 'msvc':
+                if "gfortran" in os.environ["FC"].lower():
+                    cmake_args += ['-G', 'MinGW Makefiles']
+                elif self.compiler.compiler_type == 'msvc':
                     cmake_args += ['-DCMAKE_GENERATOR_PLATFORM=x64']
                 else:
-                    cmake_args += ['-G', 'MinGW Makefiles']
+                    raise ValueError("Unable to find the system's Fortran compiler.")
```

Comment:
Question out of my naivety - is there a case where someone would be using a non-gfortran compiler (e.g. ifort), and still using MinGW?

Reply:
Well, MinGW is a collection of tools including the compilers and `make`. I've tried using the MinGW `make` with the Intel compiler, but it didn't work though I don't remember exactly why.

It's a fair question. I think the majority of cases will be covered by these if-statements, but probably not all cases.

Reply:
Sounds good to me, its probably best to tackle those fringe cases as they show up. Thanks!